asb-ballerina/messageReceiver.bal

```diff
@@ -104,7 +108,7 @@ public isolated client class MessageReceiver {
     @display {label: "Abandon Message"}
     isolated remote function abandon(@display {label: "Message"} Message message) returns Error? {
         if message?.lockToken.toString() != DEFAULT_MESSAGE_LOCK_TOKEN {
-            return abandon(self.receiverHandle, message?.lockToken.toString());
+            return abandon(self.receiverHandle, self, message?.lockToken.toString());
```

Comment:
message?.lockToken --> is lockToken optional? Is that not a mandatory field in the message?

Reply:
Yes it is mandatory to receive messages in PEEK_LOCK mode. We use same message record to provide message to ASB sender client, where the lock_token is not mandatory.